app/dashboard/dashboard.component.e2e-spec.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+var protractor_1 = require("protractor");
+describe('Dashboard', function () {
```

Comment:
__[jshint]__
* 'describe' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->